### ISSUE TITLE
colors in view mode are always yellow

### ISSUE THREAD

**Issue by notuntoward** [chetachiezikeuzor/Highlightr-Plugin#81]
In edit mode, I can highlight text in different colors, but when I switch to view mode, these highlights are always yellow.  For example:

### Edit Mode
<img width="424" alt="image" src="https://github.com/chetachiezikeuzor/Highlightr-Plugin/assets/3851891/f924bfa6-9a1a-4b94-904a-bdff5fd1db38">

### View Mode
<img width="429" alt="image" src="https://github.com/chetachiezikeuzor/Highlightr-Plugin/assets/3851891/8c01416e-9dd0-4160-8fa9-34149140a37c">

This is on Windows with the latest Highlightr at this time, and Obsidian v 1.4.16

**Comment by harsh11-lal** [comment]
hi, did you find a work around this?

**Comment by notuntoward** [comment]
No, I never did.  But I haven't revisited this in the last year, either.
